src/App.vue

```diff
@@ -78,12 +79,38 @@ export default {
 		}
 	},
 
+	computed: {
+		conversations() {
+			return this.$store.getters.conversations
+		}
```

Comment:
```suggestion
		},
		token() {
			return this.$route.params.token
		},
		currentConversation() {
			if (this.token) {
				return this.conversations[this.token]
			}
			return undefined
		}
```

Reply:
@nickvergessen now, you can watch for `currentConversation` and `token`

Reply:
You can also have another for the name :)
```suggestion
		},
		currentConversationTitle() {
			return this.currentConversation && this.currentConversation.displayName
		}
```